docs/deployment/helm.md

```diff
@@ -102,6 +102,8 @@ We have introduced the following Chart parameters.
 | `tls.enable` | bool | false | Specifies whether to use TLS for communications between components |
 | `tls.certManager` | bool | false | If enabled, requires [cert-manager](https://cert-manager.io/docs/) to be installed and will automatically create the required TLS certificates |
 | `enableNodeFeatureApi` | bool  | false | Enable the [NodeFeature](../usage/custom-resources.md#nodefeature) CRD API for communicating node features. This will automatically disable the gRPC communication.
+| `metrics.enable` | bool | false | Specifies whether to expose mtrics using prometheus |
```

Comment:
Maybe we should mentioin here (or some not below the table or somewhere) that we're supporting metrics integration with the prometheus-operator. Other prometheys deployment methods require some extra (I think?